spec/services/lifetime_usages/usage_thresholds/check_service_spec.rb

```diff
@@ -16,10 +16,10 @@
 
   def create_thresholds(subscription, amounts:, recurring: nil)
     amounts.each do |amount|
-      subscription.plan.usage_thresholds.create!(amount_cents: amount)
+      subscription.plan.usage_thresholds.create!(amount_cents: amount, organization:)
```

Comment:
Was this failing without the organization? 

I was thinking that if you do `subscription.plan.usage_thresholds.first`, nothing will actually check that the `usage_threshold`, the `plan` and the `subscription` belongs to the same organization. Am I wrong?